includes/library.php

```diff
@@ -608,68 +720,75 @@ function bu_navigation_pages_by_parent_menu_sort_cb( $a, $b ) {
 /**
  * Formats a single page for display in a HTML list
  *
- * @param $page object Page object
- * @param $html string Option HTML to place inside the list item after the page
+ * @param object $page Page object.
+ * @param mixed  $args Wordpress-style arguments (string or array).
  * @return string HTML fragment containing list item
  */
 function bu_navigation_format_page( $page, $args = '' ) {
 	$defaults = array(
-		'item_tag' => 'li',
-		'item_id' => null,
-		'html' => '',
-		'depth' => null,
-		'position' => null,
-		'siblings' => null,
+		'item_tag'     => 'li',
+		'item_id'      => null,
+		'html'         => '',
+		'depth'        => null,
+		'position'     => null,
+		'siblings'     => null,
 		'anchor_class' => '',
-		'anchor' => true,
+		'anchor'       => true,
 		'title_before' => '',
-		'title_after' => '',
-		'section_ids' => null
-		);
-	$r = wp_parse_args( $args, $defaults );
+		'title_after'  => '',
+		'section_ids'  => null,
+	);
+	$r        = wp_parse_args( $args, $defaults );
```

Comment:
Single letter variable could perhaps be refactored

Reply:
Totally, I'm planning on additional cleanup releases